docker-compose.yml

```diff
@@ -30,3 +30,12 @@ services:
       ME_CONFIG_MONGODB_URL: mongodb://${MONGODB_USERNAME}:${MONGODB_PASSWORD}@mongo:27017/
     env_file:
       - ./.env
+
+  bot:
+    image: bot
+    container_name: bot
```

Comment:
Shall we make the container/image name more specific to FBW. Will be helpful for people running multiple containers (maybe with other bots)